### ISSUE TITLE
Bug: guild unable to parse hash of run in latest (dev) version

### ISSUE THREAD

**Issue by FelixAbrahamsson** [guildai/guildai#79]
I have an operation that uses the hash of another train operation to continue training the model from that training run (I call the operation `retrain`). The model is included as a resource for the operation, and in previous versions of guild I would do the following
```
guild run retrain model=<hash>
```
to continue training from a specified model. However, in the latest versions (starting from 0.6.7.dev4 I've found) I get this error
```
train.py: error: unrecognized arguments: --model f1ea
```
which interestingly goes away if I add "model" as a dummy argument to argparse in my train.py. I'm guessing what is supposed to happen is guild should intercept the model=<hash> argument when I run the operation and not pass it to train.py, but for some reason it is not doing this in the latest releases. For the record, the retrain operation successfully loads the model from the previous operation with this dummy argument workaround.

**Comment by gar1t** [comment]
Definitely a bug- thank you for the report! This is fixed and will be available in the next pre-release.

**Comment by gar1t** [comment]
To elaborate on the underlying problem, Guild recently started to treat resource parameters formally as flags. So when you define various required resources, each resource gets a name (if not explicitly defined) which lets you set some value as a flag. For required operation, that value is the run ID.

The bug here is that Guild is trying to pass that "flag" along to the Python script. That's why you're seeing the extra `--model` option. You can see the command that Guild runs by running `guild run ... --print-cmd`. This is useful for seeing exactly what Guild is running for your script.

**Comment by gar1t** [comment]
Fix is now available in 0.7.0.
